tests/sdk/v2/driver/test_client.py

```diff
@@ -1811,6 +1811,301 @@ def test_unknown_error(
                 with pytest.raises(_exceptions.UnknownHTTPError):
                     _ = client.get_workflow_run_logs(workflow_run_id)
 
+        class TestSystemLogs:
+            @staticmethod
+            @pytest.fixture
+            def endpoint_mocker(endpoint_mocker_base, base_uri: str):
+                """
+                Returns a helper for mocking requests. Assumes that most of the tests
+                inside this class contain a very similar set up.
+                """
+
+                return endpoint_mocker_base(
+                    responses.GET,
+                    f"{base_uri}/api/workflow-run-logs/system",
+                    # Specified in:
+                    # https://github.com/zapatacomputing/workflow-driver/blob/92d9ff32189c580fd0a2ff6eec03cc977fd01502/openapi/src/resources/workflow-run-system-logs.yaml
+                    default_status_code=200,
+                )
+
+            @staticmethod
+            def test_logs_decode(
+                endpoint_mocker, client: DriverClient, workflow_run_id: str
+            ):
+                endpoint_mocker(
+                    body=resp_mocks.make_get_wf_run_system_logs_response_with_content(),
+                    match=[
+                        responses.matchers.query_param_matcher(
+                            {"workflowRunId": workflow_run_id}
+                        )
+                    ],
+                )
+
+                assert client.get_system_logs(workflow_run_id) == [
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**thought:** ‏This assert compares to a lot of log lines. This make make the test annoying to update in the future.
Do we care specificically about each line? It may be better to test for properties on these logs, like how many are there, do we see some expected lines, etc.